lib/kwtool/crawlers/schemas/keyword.ex

```diff
@@ -0,0 +1,20 @@
+defmodule Kwtool.Crawlers.Schemas.Keyword do
+  use Ecto.Schema
+
+  import Ecto.Changeset
+
+  schema "keywords" do
+    field :phrase, :string
+    field :raw_result, :string
+    field :status, :integer
+    timestamps()
+
+    belongs_to :user, Kwtool.Accounts.Schemas.User
+  end
+
+  def changeset(keyword \\ %__MODULE__{}, attrs) do
+    keyword
+    |> cast(attrs, [:phrase, :raw_result, :status, :user_id])
+    |> validate_required([:phrase, :status, :user_id])
```

Comment:
Another validation we can add is `assoc_constraint(:user)` to check if given user_id exists in users table 💡 - https://hexdocs.pm/ecto/Ecto.Changeset.html#assoc_constraint/3

Reply:
Updated! Thanks 👍